src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -1,13 +1,16 @@
 package core.mate.academy.model;
 
-/**
- * Add some custom fields that could be only in Bulldozer
- * Do not remove no-args constructor
- */
 public class Bulldozer extends Machine {
+    private Integer weight;
```

Comment:
[Prefer Primitive Types to Boxed Types](https://medium.com/geekculture/effective-java-prefer-primitive-types-to-boxed-types-ea02eb9b8c62)